### PR TITLE
DB connection made , Model added and controller added.

### PR DESCRIPTION
**Pull request by joser098**
--I've made connection to a mySQL DB. Defined using environment variables (dotnev).
In index.js You're gonna find a function called testdb that will help you to check connection with your DB.

--I've added Point_polygon model
--I've added  location_get controller
--I've added mysql2 and sequelize dependecies to package.json

### CODE REVIEW

**Review by FacundoPucheta — DISMISSED** [review]
**location_get** 
` .findAll() can be .findOne() ?`

File: src/controllers/serie_two_get.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const serie_two_get = (location, tr) => {
+    const result = [];
+    const peligro = _peligro(location, tr) // get the value that peligros's function return.
```
Comment (manduinca) [inline src/controllers/serie_two_get.js]: Please, in english, maybe: `hazard`.

File: src/controllers/serie_two_get.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const serie_two_get = (location, tr) => {
```
Comment (manduinca) [inline src/controllers/serie_two_get.js]: Can we search a more suitable name?, umm, maybe: `getUniformHazardSpectrum`.

File: src/controllers/location_get.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const { Location } = require('../db');
+
+const location_get = async (lat, long) => {
```
Comment (manduinca) [inline src/controllers/location_get.js]: `getLocation`?

File: src/models/Point_polygon.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const { DataTypes } = require('sequelize');
+
+// Export model as a function
+module.exports = (sequelize) => {
+    sequelize.define('Point_polygon', {
```
Comment (manduinca) [inline src/models/Point_polygon.js]: A better name can be `polygon_points`.

File: src/models/Point_polygon.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        id: {
+            type: DataTypes.BIGINT(11),
```
Comment (manduinca) [inline src/models/Point_polygon.js]: Why not `INTEGER`?

**Review by manduinca — CHANGES_REQUESTED** [review]
Please, I put some comments, review it.

**Review by SebaSatelier — APPROVED** [review]
looks fine, some indentation errors but nothing serious. That can be change later

File: src/db.js [hunk 6]
```diff
@@ -1 +1,19 @@
...
+    DB, DB_USERNAME, DB_PASSWORD, DB_HOST
+} = process.env;
+
+
+// Database Instance
+const sequelize = new Sequelize(DB, DB_USERNAME, DB_PASSWORD, {
+    host: DB_HOST,
+    dialect: 'mysql'
+});
+
```
Comment (manduinca) [inline src/db.js:14]: One empty line is sufficient here.

File: src/db.js [hunk 7]
```diff
@@ -1 +1,19 @@
-// conexion a DB
\ No newline at end of file
+// DB conexion
+require('dotenv').config();
+const { Sequelize } = require('sequelize');
+const {
+    DB, DB_USERNAME, DB_PASSWORD, DB_HOST
+} = process.env;
+
```
Comment (manduinca) [inline src/db.js:7]: Only one blank line, I think.

File: src/controllers/getLocations.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+const { Location } = require('../db');
+
+const getLocations = async (lat, long) => {
+    // Get lat and long by query.
```
Comment (manduinca) [inline src/controllers/getLocations.js:4]: These comments are not necessaries. Your code explain it.

File: index.js [hunk 9]
```diff
@@ -1,9 +1,23 @@
 require('dotenv').config();
 const server = require('./src/app');
+const { conn } =require('./src/db');
 
+// ###### FUNCTION TO TEST CONNECTION WITH DATABASE ######
```
Comment (manduinca) [inline index.js:5]: Do we need these comments?

**Review by manduinca — APPROVED** [review]
I leaved some commenta, I approved it.

**Pull request merged**